src/components/map/layer-hover-info.js

```diff
@@ -81,7 +113,27 @@ const EntryInfoRow = ({item, fields, data}) => {
   const displayValue = item.format
     ? getFormatter(item.format, field)(data[valueIdx])
     : parseFieldValue(data[valueIdx], field.type);
-  return <Row name={item.name} value={displayValue} />;
+  let displayDeltaValue = null;
+  if (
+    primaryData &&
+    (field.type === ALL_FIELD_TYPES.integer || field.type === ALL_FIELD_TYPES.real)
+  ) {
+    const deltaValue =
+      compareType === COMPARE_TYPES.PERCENTAGE
+        ? 1 - data[valueIdx] / primaryData[valueIdx]
```

Comment:
is this the right math?  shouldn't it be ` data[valueIdx] / primaryData[valueIdx] - 1`?

Reply:
Welp I guess I'm that bad at math haha